quarkus/config-api/src/main/java/org/keycloak/config/LoggingOptions.java

```diff
@@ -247,4 +247,13 @@ public String toString() {
             .description("Set the Syslog output to JSON or default (plain) unstructured logging.")
             .build();
 
+    // we can use SyslogConfig.CountingFraming type once https://github.com/quarkusio/quarkus/pull/48479 is present
+    public static final String SYSLOG_COUNTING_FRAMING_PROTOCOL_DEPENDENT = "protocol-dependent";
+    public static final Option<String> LOG_SYSLOG_COUNTING_FRAMING = new OptionBuilder<>("log-syslog-counting-framing", String.class)
+            .category(OptionCategory.LOGGING)
+            .expectedValues(Boolean.TRUE.toString(), Boolean.FALSE.toString(), SYSLOG_COUNTING_FRAMING_PROTOCOL_DEPENDENT)
+            .defaultValue(Boolean.FALSE.toString())
```

Comment:
This differs from the PR done for main: https://github.com/keycloak/keycloak/pull/40621

Reply:
Does that mean there will need to be a 26.3 release note describing the change in the defaulting?

Reply:
@shawkins I was thinking about it as well, but didn't provide it as it depends on the "used" point of view. Before, we didn't have any control mechanism for the Syslog counting framing as it was considered as Quarkus internals. With the PR to 26.3, we introduced the mechanism and changed defaults, which will be the same in the next Quarkus versions. So it might be perceived as some defaulting changed on the Quarkus "side", which we don't really describe. 

But I agree to write some note to the "Notable changes" to have it properly addressed. I'll add it there.